src/strokes.ts

```diff
@@ -243,8 +233,14 @@ export class Stroke extends Modifier {
 
     // Draw the rasquedo "R"
     if (this.type === Stroke.Type.RASQUEDO_DOWN || this.type === Stroke.Type.RASQUEDO_UP) {
+      const textFont = {
+        family: Tables.lookupMetric(`Strokes.text.fontFamily`),
+        size: Tables.lookupMetric(`Strokes.text.fontSize`),
+        weight: Tables.lookupMetric(`Strokes.text.fontWeight`),
+        style: Tables.lookupMetric(`Strokes.text.fontStyle`),
+      };
```

Comment:
If the `CommonMetrics` was defined with a `font` key, perhaps we could do this?

For example:
```js
FretHandFinger: {
  font: {
    family: 'Arial, sans-serif',
    size: 9,
    weight: 'bold', 
  },
},

PedalMarking: {
  font: {
    family: 'Times New Roman, serif',
    size: 12,
    weight: 'bold',
    style: 'italic',
  },
},
```

Then maybe we can do a lookupMetric('PedalMarking.font') to retrive the entire object?

Reply:
Changed to `Tables.lookupMetricFontInfo(`Strokes.text`);`

I did not follow the approach (I considered it), because I wanted to resolve the values (defaults defined a level above) at size, family... level not at font level.

With your proposal:
```js
{
 font: {
    family: 'Times New Roman, serif',
    size: 12,
    weight: 'bold',
    style: 'italic',
 },

 FretHandFinger: {
  font: {
    family: 'Arial, sans-serif',
    size: 9,
    weight: 'bold', 
   },
},
```
`FretHandFinger` would not be `italic`.